testcases/extensions-map.json

```diff
@@ -0,0 +1,18 @@
+{
+    "specversion": "0.4-wip",
+    "id": "123",
+
+    "source": "https://cloudevents.io/extensions-map",
+    "type": "io.cloudevents.extensions-map-test",
+
+    /***
+     * EXTENSIONS, with a map value
+     */
+    "this0is0ok": {
+        "\": "ok",
+        // Empty string **is** allowed by RFC7159.
+        "": {
+            "\u005c": "17"
```

Comment:
you show `17` as a string here but as a number in testcases/extensions-map.http - should they match?

Reply:
Integers and Strings have this problem for unknown extension schemas, I believe.

I.e. given an HTTP header:

```
CE-Foobar: 17
```

Does this render as:

```
{
  ...
  "foobar": 17
}
```
or
```
{
  ...
  "foobar": "17"
}
```

Reply:
yes that's true - I think it'll show up as a string though since I think unknown attributes are kept as string.  But, was that the point of the test? I only commented on it because I thought you were trying to be consistent - if not then never mind.